src/apps/routes/schemas/app.clj

```diff
@@ -261,8 +262,9 @@
    :name (describe String "The App Category's name")})
 
 (defschema AppDetailsTool
-  (assoc Tool
-    :id (describe String "The tool identifier.")))
+  (merge Tool
+         {:id                       (describe String "The tool identifier.")
+          (optional-key :container) {:image (dissoc containers/Image :id)}}))
```

Comment:
Thanks for the suggestion. I think I'll probably make it before merging this request just so that I don't forget to do it later.